apps/emqx_durable_storage/src/emqx_ds_replication_layer.erl

```diff
@@ -279,22 +282,19 @@ make_delete_iterator(DB, Stream, TopicFilter, StartTime) ->
     case ra_make_delete_iterator(DB, Shard, StorageStream, TopicFilter, StartTime) of
         {ok, Iter} ->
             {ok, #{?tag => ?DELETE_IT, ?shard => Shard, ?enc => Iter}};
-        Err = {error, _} ->
-            Err
+        Error = {error, _, _} ->
```

Comment:
The contract for `delete` functions seems to be still not updated and expect 2-sized tuples 😓